docs/apps/components/number.md

```diff
@@ -0,0 +1,193 @@
+    2. **Fields** refers to a **component ID** or a **list of component IDs**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. Before using **this.show(fields)**, make sure the component is **hidden**. This can be done using **this.hide(fields)** or by **enabling** the **Hidden checkbox** in the Component Attribute panel.
+    4. Before using **this.enable(fields)**, make sure the component is **disabled**. This can be done using **this.disable(fields)** or by **enabling** the **Disabled checkbox** in the Component Attribute panel.
+
+#### Steps to use the methods for the Page Builder components
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action').
+2. Write the method/code as shown in the **Example** below.
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/check1.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps mentioned above, under Steps to use the methods for the Page Builder components
+          * ```
+            this.addClassName('number_kzdn0gvb', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('number_kzdn0gvb', 'abc')
+        ```
+    
+    3. `this.getValues()`
+        ```js
+        var data = this.getValues();
+        console.log(data);
+        ```
+    
+    4. `this.hide(fields)`
+        ```js
+        var fields= ['number_kzdn0gvb']
+        this.hide(fields)
+        ```
+    
+    5. `this.show(fields)`
+        ```js
+        var fields= ['number_kzdn0gvb']
+        this.show(fields)
+        ```
+    
+    6. `this.disable(['fields'])`
+        ```js
+        this.disable(['number_kzdn0gvb'])
+        ```
+    
+    7. `this.enable(['fields'])`
+        ```js
+        this.enable(['number_kzdn0gvb'])
+        ```
+    
+    8.  `this.getValue('fieldName')`
+        ```js
+        var numbername = this.getValue('number_kzdn0gvb');
+        console.log('getValue', numbername);
+        ```
+    
+    9.  `this.setData(Value)`
+        ```js
+        this.setData(
+         {"number_kzdn0gvb": "2024",});
+        ```
+    
+    10.  `this.getComponent('component_ID')`
+        ```js
+        var numbername = this.getComponent('number_kzdn0gvb');
+        console.log('getComponent', numbername);
+        ```
+
+!!! info Add multiple classes
+    Use the following syntax to add multiple classes to a component:
+    ```js
+    this.addClassName('componentID', 'class1', 'class2')
```

Comment:
This syntax is not working, only the first class gets added while the others are ignored